src/environments/jest.ts

```diff
@@ -12,19 +12,27 @@ export const getJestEnvironment: TestEnvironmentGetter = () => {
   ) {
     return undefined;
   }
+  let afterEachCallback = () => {};
+  let beforeEachCallback = () => {};
```

Comment:
Hmm, looks like CI isn't running on this repo. I'll get it set back up.

```suggestion

  /* eslint-disable @typescript-eslint/no-empty-function */
  let afterEachCallback = () => {};
  let beforeEachCallback = () => {};
  /* eslint-enable @typescript-eslint/no-empty-function */
```